src/test/banman_tests.cpp

```diff
@@ -8,16 +8,16 @@
 #include <streams.h>
 #include <test/util/logging.h>
 #include <test/util/setup_common.h>
+#include <test/util/time.h>
 #include <util/readwritefile.h>
 
-
 #include <boost/test/unit_test.hpp>
 
 BOOST_FIXTURE_TEST_SUITE(banman_tests, BasicTestingSetup)
 
 BOOST_AUTO_TEST_CASE(file)
 {
-    SetMockTime(777s);
+    ElapseTime elapse_time{777s};
```

Comment:
I find it non-intuitive that we have constructor and operator doing different things, i.e. why the absolute setter:
```C++
    ElapseTime elapse_time{777s};
```
is different from the relative one:
```C++
    ElapseTime elapse_time{};
    elapse_time(777s);
```